pkg/oci/load_balancer_spec.go

```diff
@@ -30,19 +30,29 @@ import (
 	"github.com/oracle/oci-cloud-controller-manager/pkg/oci/util"
 )
 
+// SSLK8SSecret is a structure containing the data about a K8S secret required
+// to store SSL information required for BackendSets and Listeners
+type SSLK8SSecret struct {
+	CACert     *string
```

Comment:
I'd avoid user `*string` (unless there's a good reason?)

Reply:
No, nit really, I wanted to keep it in line with the loadbalancer.CertificateDetails format